### PR TITLE
refactor: change fixed header to sticky

### PR DESCRIPTION
**Pull request by rafirm29**
## Summary
this is a test for flicker framic solution. Some solutions that will be tested in this PR such as:
- Change header from fixed to sticky (to make scrolling smoother)
- Add webkit style for safari (got from online article regarding Safari scrolling bug)

## Files changed
`components/Header/Header.tsx`
`public/scss/components/Header.module.scss`
`public/scss/components/ProductDetail.module.scss`
Other files were merged from source branch.

## Build status
image
✔ Build OK on local

### CODE REVIEW

File: pages/[lng]/product/[slug].tsx [hunk 1]
```diff
@@ -1,43 +1,37 @@
 /* library package */
-import { FC, ReactElement } from 'react'
-import {
-  useI18n,
-  getProductDetail,
-  useAuthToken
-} from '@sirclo/nexus'
-import { LazyLoadComponent } from 'react-lazy-load-image-component'
+import { FC, ReactElement, ReactNode } from 'react';
```
Comment (mayun19) [inline pages/[lng]/product/[slug].tsx:2]: bila ada 3 variable atau lebih dibuat line ke bawah

File: pages/[lng]/product/[slug].tsx [hunk 2]
```diff
@@ -1,43 +1,37 @@
...
+import { useI18n, getProductDetail, useAuthToken } from '@sirclo/nexus';
```
Comment (mayun19) [inline pages/[lng]/product/[slug].tsx:3]: idem

File: pages/[lng]/products/[slug].tsx [hunk 3]
```diff
@@ -1,41 +1,41 @@
...
-import { GetServerSideProps, InferGetServerSidePropsType } from 'next'
+import { FC, useState } from 'react';
+import { GetServerSideProps, InferGetServerSidePropsType } from 'next';
 import {
   useI18n,
   useAuthToken,
   TemplateFeatures,
-  FeaturesType
-} from '@sirclo/nexus'
+  FeaturesType,
```
Comment (mayun19) [inline pages/[lng]/products/[slug].tsx:8]: coma nya ngaruh ga klo di hapus?

File: pages/[lng]/product/[slug].tsx [hunk 4]
```diff
@@ -1,43 +1,37 @@
...
+import { LazyLoadComponent } from 'react-lazy-load-image-component';
 /* library template */
-import { useBrand } from 'lib/useBrand'
-import { NextPageWithLayout } from 'lib/commonTypes'
+import { useBrand } from 'lib/useBrand';
```
Comment (mayun19) [inline pages/[lng]/product/[slug].tsx:6]: petik 2 tanpa ; biasanya

**Review by mayun19 — CHANGES_REQUESTED** [review]

File: pages/_app.js [hunk 5]
```diff
@@ -1,19 +1,19 @@
-import "@brainhubeu/react-carousel/lib/style.css";
-import "react-toastify/dist/ReactToastify.css";
-import "public/scss/main.scss";
+import '@brainhubeu/react-carousel/lib/style.css';
```
Comment (mayun19) [inline pages/_app.js:1]: ini juga raf

**Review by mayun19 — CHANGES_REQUESTED** [review]

**Review by mayun19 — APPROVED** [review]

**Review by DiebrataA — APPROVED** [review]

**Review by virginiaritonga — APPROVED** [review]

**Pull request merged**